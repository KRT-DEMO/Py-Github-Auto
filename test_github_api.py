import requests

GitHub_API = 'https://api.github.com'
User_Repos_Resource = '/user/repos'
headers = {'Accept': 'application/vnd.github.v3+json'}
#bearertoken = '3f275dd3fa9803072dd1299175f1e10079d0124a'
bearertoken = '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'

response = requests.get(GitHub_API, headers=headers, auth=('Authorization', bearertoken))

print(response.status_code)
print(response.content)