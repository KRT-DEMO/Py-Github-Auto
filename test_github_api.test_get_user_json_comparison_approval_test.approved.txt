{
    "login": "KRT-DEMO",
    "id": 95260371,
    "node_id": "U_kgDOBa2O0w",
    "avatar_url": "https://avatars.githubusercontent.com/u/95260371?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/KRT-DEMO",
    "html_url": "https://github.com/KRT-DEMO",
    "followers_url": "https://api.github.com/users/KRT-DEMO/followers",
    "following_url": "https://api.github.com/users/KRT-DEMO/following{/other_user}",
    "gists_url": "https://api.github.com/users/KRT-DEMO/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/KRT-DEMO/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/KRT-DEMO/subscriptions",
    "organizations_url": "https://api.github.com/users/KRT-DEMO/orgs",
    "repos_url": "https://api.github.com/users/KRT-DEMO/repos",
    "events_url": "https://api.github.com/users/KRT-DEMO/events{/privacy}",
    "received_events_url": "https://api.github.com/users/KRT-DEMO/received_events",
    "type": "User",
    "site_admin": false,
    "name": null,
    "company": null,
    "blog": "",
    "location": null,
    "email": null,
    "hireable": null,
    "bio": null,
    "twitter_username": null,
    "public_gists": 0,
    "followers": 1,
    "following": 0,
    "created_at": "2021-11-30T01:04:29Z",
    "private_gists": 0,
    "total_private_repos": 0,
    "owned_private_repos": 0,
    "collaborators": 0,
    "two_factor_authentication": false,
    "plan": {
        "name": "free",
        "space": 976562499,
        "collaborators": 0,
        "private_repos": 10000
    }
}
